quickwit-metastore/src/metastore/file_backed_metastore/file_backed_index.rs

```diff
@@ -387,8 +400,93 @@ impl FileBackedIndex {
         Ok(true)
     }
 
-    /// Resets the checkpoint of a source. Returns whether a mutation occurred.
```

Comment:
This is the only function with the docs, either we put that everywhere or we don't.

Reply:
lol, put it everywhere then, rather than removing this line. How is removing some relevant docs a good thing?

Reply:
Yes.... my comment seems strange but... the doc is already in the `mod.rs` file. We can choose to repeat the docs everywhere. I don't know what's the good practice when you are just implementing a trait.